tests/migrations/test_operations.py

```diff
@@ -1126,10 +1126,14 @@ def test_remove_field_non_nullable(self):
             operation.database_forwards("test_rmflnn", editor, project_state, new_state)
 
         # And test reversal
-        with self.assertRaises(IntegrityError):
-            with transaction.atomic():
-                with connection.schema_editor() as editor:
-                    operation.database_backwards("test_rmflnn", editor, new_state, project_state)
+
+        # MySQL populates NOT NULL cols with implicit defaults instead of raising an error.
+        if connection.vendor != 'mysql':
```

Comment:
Do any other DBs have this behavior such that it would be worth adding a feature flag?